lib/global_uid/base.rb

```diff
@@ -46,8 +46,6 @@ def self.init_server_info
       increment_by = GlobalUid.configuration.increment_by
       connection_timeout = GlobalUid.configuration.connection_timeout
 
-      raise "You haven't configured any id servers" if id_servers.nil? or id_servers.empty?
```

Comment:
hm... I am a bit torn about this. This is a configuration error, vs the other one, which can be caused by infra failure.

Reply:
Fair point. I'll update the configuration class so it validates `id_servers.length > 0`